mlir-tensorrt/compiler/include/mlir-tensorrt/Compiler/StableHloToExecutable.h

```diff
@@ -189,6 +189,13 @@ class StableHloToExecutableTask
                                const StableHLOToExecutableOptions &options);
 };
 
+// TODO: Figure out if we want to register in a different way.
+static const bool registeredStableHloToExecutable =
```

Comment:
Lots of ways that we could implement the registration mechanism. I went with a straightforward approach here but we can do something better/cleaner if preferred.

Reply:
this definitely shouldn't be in a header file, that will break some build configs.

We should just do it like this:

In the header file do

```
void registerStableHloToExecutableTask();
```

In CPP file:

```C++
void compiler::registerStableHloToExecutableTask() {
   CompilerClient::registerOption(
        "stable-hlo-to-executable",
        optionsCreateFromArgs<StableHLOToExecutableOptions,
                              StableHloToExecutableTask>);
}
```

Now you need to invoke `registerStableHloToExecutableTask` somewhere. The easiest thing to do is just grep for `registerStablehloClusteringPipelines` and call `registerStableHloToExecutableTask` there as well.